db/migrations/20221005071834_create_social_types_table.sql

```diff
@@ -0,0 +1,8 @@
+-- migrate:up
+CREATE TABLE social_types(
+id int not null auto_increment primary key,
+name varchar(100) not null unique key
+);
```

Comment:
```suggestion
CREATE TABLE social_types(
  id int not null auto_increment primary key,
  name varchar(100) not null unique key
);
```
다른 migration 파일들과 마찬가지로 indentations을 맞춰주면 좋을 것 같아요!

Reply:
인덴테이션을 탭으로 눌러서 통일하였는데 2칸, 4칸으로 통일되지 않습니다. 혹시 이유가 있을까요? 일단은 탭 한번만 눌러서 다시 피알합니다. (응수님과 저 모두 탭 prettier Indentation 2로 통일한 상태입니다.)